packages/salesforcedx-apex-debugger/package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "salesforce",
   "license": "BSD-3-Clause",
   "engines": {
-    "vscode": "^1.32.0"
+    "vscode": "^1.40.0"
```

Comment:
Updating the extensions to be used on VSCode versions 1.40 and above, most users are already on those versions.